reactnative/RNPlayground/app/screens/PlaygroundScreen.tsx

```diff
@@ -1,76 +1,108 @@
-import React, { FC } from "react"
+import * as React from "react"
```

Comment:
This is fine but since we have `allowSyntheticDefaultImports: true` in `tsconfig.json` we don't need this change.

I actually found that this wildcard import is best practise (they exist in RN?) 😱 and shared another interesting guidelines below
https://github.com/facebook/react/pull/18102
https://github.com/facebook/react-native-website/blob/main/STYLEGUIDE.md

Reply:
There's a difference in behaviour (at least on my VSCode so it might be my config) with autocomplete. When I start writing a function from react such as `useEffect` then autocomplete behaves differently depending on the import type: 

In case of `import * as React ...` autocomplete will update the function call to `React.useEffect`
In case of `import React ...` the autocomplete will add another import `{useEffect}` and not change the function call

The `import *` is definitely more verbose but at least for now it makes it easier for me to understand where the code comes from even if it is probably not the correct JS approach